cmd/ktn-linter/cmd/.ktn-linter.yaml

```
@@ -0,0 +1,1289 @@
+    enabled: true
+
+  # KTN-VAR-WEAKRAND [WARN]
+  # Detects math/rand usage in security context
+  KTN-VAR-WEAKRAND:
+    enabled: true
+
+  # KTN-VAR-WGGO [INFO]
+  # Detects wg.Add(1)+go func()+defer wg.Done() - use wg.Go() (Go 1.25+)
+  KTN-VAR-WGGO:
+    enabled: true
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)